app/models/metasploit/cache/module/relationship.rb

```diff
@@ -7,19 +7,24 @@ class Metasploit::Cache::Module::Relationship < ActiveRecord::Base
   # Associations
   #
 
-  # @!attribute ancestor
-  #   The {Metasploit::Cache::Module::Ancestor} whose {Metasploit::Cache::Module::Ancestor#real_path file} defined the ruby Class or ruby Module.
-  #
-  #   @return [Metasploit::Cache::Module::Ancestor]
+  # The {Metasploit::Cache::Module::Ancestor} whose {Metasploit::Cache::Module::Ancestor#relative_path file} defined the
+  # ruby Class or ruby Module.
   belongs_to :ancestor, class_name: 'Metasploit::Cache::Module::Ancestor', inverse_of: :relationships
 
-  # @!attribute descendant
-  #   The {Metasploit::Cache::Module::Class} that either has the Module in {#ancestor} mixed in or is the Class in {#ancestor}.
-  #
-  #   @return [Metasploit::Cache::Module::Class]
+  # The {Metasploit::Cache::Module::Class} that either has the Module in {#ancestor} mixed in or is the Class in
+  # {#ancestor}.
   belongs_to :descendant, class_name: 'Metasploit::Cache::Module::Class', inverse_of: :relationships
 
   #
+  # Attributes
+  #
+
+  # @!method ancestor_id
```

Comment:
Is there a reason these aren't `@!attribute`s? Using `@!method` will not create a setter.